docker-compose.yml

```diff
@@ -2,8 +2,11 @@ name: saas-backend
 
 services:
   api:
+    platform: linux/amd64
     env_file:
       - .env
+    environment:
+      - VOCDONI_MONGOURL=mongodb://root:vocdoni@mongo:27017/
```

Comment:
@lucasmenendez doesn't it make sense to import this from .env just as with `VOCDONI_PORT`

Reply:
No, because it is hardcoded into the other compose services.